.github/workflows/auto-updates.yml

```diff
@@ -50,9 +50,14 @@ jobs:
     - name: Render templates
       run: python -Wd -m wikidict ${{ matrix.locale }} --render
 
-    - name: Convert to dictionaries
+    - name: Convert to dictionaries (DictFile and Kobo)
       run: python -Wd -m wikidict ${{ matrix.locale }} --convert
 
+    - name: Generate dictionary (StarDict)
+      run: |
+        pyglossary --read-format Dictfile --write-format Stardict --no-progress-bar --no-color data/${{ matrix.locale }}/dict-${{ matrix.locale }}.df ifo
+        zip dict-${{ matrix.locale }}.zip ifo.dict.dz ifo.idx ifo.ifo
```

Comment:
ifo.dict.dz ? really ?
I used 
```
python3 main.py ../ebook-reader-dict/data/fr/dict-fr.df dict-fr.ifo
```

and I got 

```
dict-fr.dict	dict-fr.idx	dict-fr.ifo	dict-fr.syn
```

Reply:
And I got dict.dz and no .syn :)

Reply:
Maybe local results are bad (for whatever reason). But ideally pyglossary generates de final Zip, it would ease things.

Reply:
We could do something like:
```shell
$ pyglossary --read-format Dictfile --write-format Stardict --no-progress-bar --no-color data/${{ matrix.locale }}/dict-${{ matrix.locale }}.df dict-data.ifo
$ zip dict-${{ matrix.locale }}.zip dict-data.*
```
But I am not sure of the Zip filenames then. Do you know if names are important or just the file extension maybe?